packages/example/components/chains/ethereum/utils.ts

```
@@ -1,6 +1,7 @@
-function parseChainId(chainId: string | undefined): number {
+export function parseChainId(chainId: string | undefined): number {
   return parseInt(chainId?.startsWith('0x') ? chainId ?? '0' : `0x${chainId}`, 16);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)